src/platforms/common/crons/index.mdx

```diff
@@ -136,19 +136,19 @@ format and the `sentry.io/api` format will be deprecated.
 
 <SignInNote />
 
-```bash {tabTitle: cURL}
-# Upsert monitor with a check-in
-curl -X POST \
-    'https://sentry.io/api/0/organizations/___ORG_SLUG___/monitors/<monitor_slug>/checkins/' \
-    --header 'Authorization: DSN ___PUBLIC_DSN___' \
+````bash {tabTitle: cURL}
+SENTRY_INGEST="https://___ORG_INGEST_DOMAIN___"
+SENTRY_CRONS="${SENTRY_INGEST}/api/___PROJECT_ID___/cron/<monitor_slug>/___PUBLIC_KEY___/"
+
+# 🟡 Notify Sentry your job is running:
+curl -X POST "${SENTRY_CRONS}"
     --header 'Content-Type: application/json' \
     --data-raw '{"monitor_config": {"schedule": "0 * * * *", "schedule_type": "crontab"}, "status": "in_progress"}'
-```
+j``
```

Comment:
not sure if intended, but seems like some mistypings here with the `j` and backticks

Reply:
lol oops